ERCS/eip-draft-request-permissions-from-wallets.md

```diff
@@ -0,0 +1,436 @@
+---
```

Comment:
```suggestion
---
eip: 7715
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.